src/server/frontend/render.js

```diff
@@ -64,23 +69,23 @@ function fetchComponentData(dispatch, req, {components, location, params}) {
   // Because redux-promise-middleware always returns fulfilled promise, we have
   // to detect errors manually.
   // https://github.com/pburtchaell/redux-promise-middleware#usage
-  return Promise.all(promises).then(results => {
-    results.forEach(result => {
-      if (result.error)
-        throw result.payload;
-    });
+  const results = await Promise.all(promises);
```

Comment:
BTW https://github.com/facebook/regenerator/commit/ca06ad55e1e1fd840c2ffc31c1391b85a8dd8459. Rejected, but likely to be added latter.

Then, we could just do:
```
const results = await promises
```
?

In that case, I'd also rename `promises` to something more obvious, like `fetchActions`.